src/components/Papers/PapersList.jsx

```diff
@@ -29,70 +32,60 @@ import {
 import { getReferencedBy } from 'cozy-client'
 import { CONTACTS_DOCTYPE } from 'src/doctypes'
 import { buildPaperslistByContact } from 'src/helpers/buildPaperslistByContact'
+import { usePapersDefinitions } from 'src/components/Hooks/usePapersDefinitions'
 
 const useStyles = makeStyles({
   root: { textIndent: '1rem' }
 })
+const backButtonStyle = { marginRight: '0.25rem' }
```

Comment:
If we have margin that didn't match our current "margins" we should ask us if this is really what we want. 

Our "margins" are defined here: 0.5 / 1 / 2 / ... 
https://docs.cozy.io/cozy-ui/styleguide/section-utilities.html#kssref-utilities-margins